bin/alluxio-start.sh

```diff
@@ -340,22 +340,22 @@ main() {
     exit 1
   fi
 
+  if [[ "${killonstart}" != "no" ]]; then
```

Comment:
does the comment need to be changed: https://github.com/aaudiber/alluxio/blob/3eff009cd3fa8ed36bce0863cd276c48f559f03b/bin/alluxio-start.sh#L46

Reply:
updated, thanks